specreduce/tests/test_extinction.py

```diff
@@ -58,12 +61,19 @@ def test_missing_extinction_unit():
     """
     wave = np.linspace(0.3, 2.0, 50)
     extinction = 1. / wave
-    ext = AtmosphericExtinction(extinction=extinction, spectral_axis=wave * u.um)
-    assert(len(ext.extinction_mag) > 0)
-    assert(len(ext.transmission) > 0)
+    with pytest.warns(AstropyUserWarning):
+        ext = AtmosphericExtinction(extinction=extinction, spectral_axis=wave * u.um)
+
+    assert len(ext.extinction_mag) > 0
+    assert len(ext.transmission) > 0
 
 
 def test_transmission_model():
+    """
+    Test creating of default atmospheric transmission model
+    """
     ext = AtmosphericTransmission()
-    assert(len(ext.extinction_mag) > 0)
-    assert(len(ext.transmission) > 0)
+    assert len(ext.transmission) > 0
+
+    with pytest.warns(RuntimeWarning):
```

Comment:
Huh, that's weird. Why would the `len()` give RuntimeWarning here?

Reply:
the `extinction_mag` property in that case is doing a `log10` operation that generates that warning because there are wavelengths with 0 transmission.

Reply:
I wonder if this means this warning ignore needs to be moved into the property or the whole code needs some refactoring but I guess that is out of scope here. Thanks for the clarification!

Reply:
yeah, it's a bigger question how to best handle zero/negative values in the presence of logarithms. i think, though, one will always want a warning when an operation on an array of valid numbers generates an array with some `inf`/`nan` values. so it's a matter of whether to leave it as the original `numpy` warning or wrap it into a custom one.